contracts/QuestFactory.sol

```diff
@@ -150,6 +151,7 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
 
         amountMintedForQuestId[questId_] += amount_;
         addressMintedForQuestId[questId_][msg.sender] = true;
+        numberMintedForQuestId[questId_]++;
```

Comment:
Isn't this a duplicate of `amountMintedForQuestId`?

Reply:
Kind of but not really, but it is a good question on if we should use `amountMintedForQuestId` instead of `numberMintedForQuestId`.